__tests__/api.test.js

```diff
@@ -22,19 +24,43 @@ describe.only('GET /api/topics', () => {
         .then(({ body }) => {
         const topics = body.topics;
         expect(Array.isArray(topics)).toBe(true);
-        expect(topics.length).toBe(3)
         topics.forEach((topic) => {
```

Comment:
You should definitely be checking the length of the array that is passed when using a forEach, as it could produce a false positive if not

Reply:
do you mean line 25(that has been removed) should be included?
 i think i added it to the wrong branch so deleted it and updated the correct branch then merged
 it is included on the main branch